certbot/renewal.py

```diff
@@ -408,6 +409,16 @@ def handle_renewal_request(config):
         try:
             if renewal_candidate is None:
                 parse_failures.append(renewal_file)
+            elif lineage_config.certname  and cli.set_by_cli('autorenew'):
+                # Update renewal config file and exit.
+                symlinks = dict((kind, renewal_candidate.configuration[kind])
+                                for kind in storage.ALL_FOUR)
+                storage.update_configuration(lineagename,
+                        renewal_candidate.archive_dir, symlinks, lineage_config)
+                disp.notification("{} auto renewal for {}".format(
+                    "Enabled" if config.autorenew else "Disabled",
+                    lineagename))
+                sys.exit(0)
```

Comment:
moving the check to `storage` will automatically handle catching certs we don't renew, so shouldn't need to `sys.exit`.

Reply:
Removed `sys.exit`.

Due to this, when the user does

```bash
# certbot renew --cert-name foo.bar.pub  --autorenew
```
certbot  will enable autorenew (set the `autorenew = True` in renewal config file) and will try to renew the certificate for `foo.bar.pub`. Is this the desired behavior or do we want certbot to 
 just enable autorenew and exit?